perun-openapi/openapi.yml

```diff
@@ -7800,6 +7800,20 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/membersManager/getSponsors:
```

Comment:
Do we want only this version of the method or we want to support passing ``attrNames`` - since we retrieve RichUsers exactly to get their attributes too. And what about combination with ``vo, extSourceName, extLogin`` and again optionally ``attrNames``?
You can add more versions of the same method using same pattern as in below method, where you append ``/someString`` to the URL.

Reply:
Second version of this method was added. For optional parameter `attrNames` I used already defined paramter `attrNamesOptional`.